### DISCUSSION TITLE
Create same account in second portal based on username e-mailadres

### DISCUSSION THREAD

**Discussion by wbonekamp** [dnnsoftware/Dnn.Platform#5472] [Q&A]
## Description of bug
We have a DNN instance with 2 portals (1, 2). Portal 1 has 500 users that login with e-mailadres (username).
All the 500 users should also be available in portal 2. Unfortunately they can't login to portal 2.
I tried to add the users to portal 2 but this isn't possible because e-mailadres is the username and this should be unique.

I already did check the settings in config and it's false: requiresUniqueEmail="false"

Why can't we add/create the same user in portal 2

## Steps to reproduce
List the precise steps to reproduce the bug:
1. Go to portal 1 
2. Got to 'Manage' - 'Users'
3. Click on 'Add users"
4. Add user with username name@domain.nl
5. User is created
6. Go to portal 2
7. Got to 'Manage' - 'Users'
8. Click on 'Add users"
9. Add the same user with username name@domain.nl
10. Validation error: "the username is already in use"

## Current behavior
I already did check the settings in config and it's false: requiresUniqueEmail="false"
Looks like this is an bug/issue in the latest version of DNN because in other version this was possible.

I know you can use Site Groups.. but we need the user accounts to be created in portal 2

## Expected behavior
this should be possible when setting requiresUniqueEmail="false"

## Screenshots
![theusernameisalreadyinuse](https://user-images.githubusercontent.com/7567981/211321889-6379ac36-18eb-474b-a268-37254901ed06.PNG)

## Affected version
* [ ] 09.11.00 release candidate
* [ ] 09.10.02 release candidate
* [ ] 09.10.01 latest supported release

**Comment by sleupold** [comment]
you need to use same username and password to grant a user access to a second portal.
Another option would be using a website group.

**Comment by wbonekamp** [comment]
using a website group it the same as add record to the userportals table? right?

**Comment by sleupold** [comment]
In a website group, current and future users become automatically member of all sites.
SQL will affect existing users only and you need to add records (at least) to userportals, userroles and userprofile tables.

**Comment by valadas** [comment]
Unfortunatelly, requirement for unique usenames is instance-wide. I'll move this to a discussion for now but if anything actionable comes back from this discussion, I'll move it back to an issue...

**Comment by mitchelsellers** [comment]
As others have mentioned this is by design.  There is a single users table, users have to be either added to the UserPortals table or be part of a portal group.

Users can register with the same username/password on other portals.

**Comment by wbonekamp** [comment]
exactly using the same username and password did the trick. Thanks!
